flutter/lib/src/utils/multi_view/html_multi_view_helper.dart

```diff
@@ -0,0 +1,11 @@
+import 'dart:js' as js;
+import 'multi_view_helper.dart';
+
+MultiViewHelper multiViewHelper() => HtmlMultiViewHelper();
+
+class HtmlMultiViewHelper implements MultiViewHelper {
+  @override
+  bool isMultiViewEnabled() {
+    return "flutter-view" == js.context['__flutterState'][0].toString();
```

Comment:
does this have to be in combination with the `SentryWidget`?

I'm also wondering whether this is deterministic throughout multiple dart versions. it would not be ideal if this is a false positive in some cases and we disable the non-multiview supported integrations and features

Reply:
I also was not quiet happy with my solution, therefore I changed it, and check the `implicitView` now.

what do you mean with "does this have to be in combination with the SentryWidget?"? You can use this helper without the `SentryWidget`. But currently this helper is only used in the `SentryWidget`, because there will be `SentryScreenshotWidget` and the `SentryUserInteractionWidget` instantiated.

Reply:
@buenaflor  I also found out, that the `PlatformDispatcher.instance.implicitView` feature was introduced in `Flutter 3.10` (May 2023). So this will not work with our backward compatibility back to `Flutter 3.0.0`

Reply:
as we discussed this will still work with try/catch dynamic and falling back to false.